solutions/sequences/TwoSum.cpp

```diff
@@ -0,0 +1,30 @@
+// https://leetcode.com/problems/two-sum/
+
+#include <vector>
+#include <unordered_map>
+
+class Solution {
+public:
+    std::vector<int> twoSum(std::vector<int>& nums, int target) {
+        std::vector<int> result;
```

Comment:
looks good, there are also followup questions like three sum. I would recommend u to visit the later on.